dev/ci/ci-deriving.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+ci_dir="$(dirname "$0")"
+. "${ci_dir}/ci-common.sh"
+
+install_ssreflect
```

Comment:
This is not needed I think; @SkySkimmer , should we just remove these functions and the couple of users from CI? I'm gonna try a PR.